angularjs-1.5-basics/src/app/angular-1.5-basics-main.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
```

Comment:
__[jshint]__
* Use the function form of "use strict".

<!-- Comment by [SideCI](https://sideci.com) -->